### PR TITLE
Fix Errorf arguments in tests

### PR DESCRIPTION
**Pull request by soltysh**
**What type of PR is this?**
/kind cleanup
/kind failing-test

**What this PR does / why we need it**:
Fixes arguments in `Errorf` invocation.

**Special notes for your reviewer**:
/assign @deads2k @sttts 

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/helpers_test.go [hunk 1]
```diff
@@ -272,10 +272,10 @@ func TestSetCRDCondition(t *testing.T) {
 		}
 		for i := range tc.expectedcrdCondition {
 			if !IsCRDConditionEquivalent(&tc.expectedcrdCondition[i], &crd.Status.Conditions[i]) {
-				t.Errorf("%v expected %v, got %v", tc.name, tc.expectedcrdCondition, crd.Status.Conditions)
+				t.Errorf("%v expected %v, got %v", tc.name, tc.expectedcrdCondition[i], crd.Status.Conditions)
```
Comment (liggitt) [inline staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/helpers_test.go]:
```suggestion
				t.Errorf("%v expected %v, got %v", tc.name, tc.expectedcrdCondition[i], crd.Status.Conditions[i])
```

**Comment by liggitt** [comment]
one nit, lgtm otherwise

**Comment by soltysh** [comment]
@liggitt updated

**Review by jpbetz — APPROVED** [review]
/lgtm

**Pull request merged**